src/tree/SubscriptionTreeItem.ts

```diff
@@ -30,8 +30,6 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         return this.cache.resourceGroups;
     }
 
-    public readonly childTypeLabel: string = localize('resourceGroup', 'Resource Group');
```

Comment:
The .gif still shows "Resource Group" when you select it.  Was that screen grab made before this change?  I can't remember what it says by default.

Reply:
I fixed it after I noticed it in the gif 😄